dispatcher/backend/src/routes/schedules/schedule.py

```diff
@@ -382,6 +382,9 @@ def post(self, schedule_name: str, token: AccessToken.Payload, session: so.Sessi
         )
         clone.durations.append(duration)
 
-        session.flush()
+        try:
+            session.flush()
+        except IntegrityError:
+            raise BadRequest("Schedule name already exists")
```

Comment:
I find this a bit risky, I'm afraid we might have other integrity constraints in the future and might forget to alter this message.

At least I would log a warning with the exact integrity issue so that we can discover this in the log, otherwise it will be difficult to debug.

Or I would analyze IntegrityError, check the issue is about the schedule name, raise this BadRequest only in this situation ;  and raise a more generic IntegrityError for other situations (which do not exist today).